### ISSUE TITLE
error LNK 2001 when building "ReconstructMesh"

### ISSUE THREAD

**Issue by lzn1273180880** [cdcseacave/openMVS#716]
Hi! Sorry to interrupt you, I encountered an error when I was building openMVS according to `BUILD.md`, all the dependencies are installed using `vcpkg`.
![image](https://user-images.githubusercontent.com/94612909/144417096-9198169f-c828-430f-9be5-31f2fd46ff46.png)
But when I tried to build `.sln`  via `Virtual Studio 16 2019`, everything was working properly except `ReconstructMesh` and error was shown as the screenshot below.
![image](https://user-images.githubusercontent.com/94612909/144417656-932ac2c5-b8ca-4f40-8129-6fad24810cce.png)
Due to that the result is I cannot find "ReconstructMesh.exe" in my build folder, and the command was not found.
![image](https://user-images.githubusercontent.com/94612909/144417814-f9c4e2e8-9e85-4605-bd1a-e3eb983a6fdc.png)
It would be vital for me if you have any hints on this.
Thank you.

**Comment by lzn1273180880** [comment]
Please reply a little quickly if it is possible.Thx.

**Comment by lzn1273180880** [comment]
Sorry for the disturbance, problem solved by modifying the define  `BOOST_NO_EXCEPTIONS` file in header file `types.`

**Comment by lzn1273180880** [comment]
Sorry for coming again, I found that the openMVS doesn't seem to run normally after a calculation. 
The first calculation was nice, I got a compelling result `.ply`, but the second time when I tried to compute another different `scene.mvs` with another dataset, the calling `.\DensifyPointCloud.exe scene.mvs` would run so quickly that the next calling `.\ReconstructMesh ` would be failed.
How can I solve this problem?
Any hints on this?
Thanks a lot.
